docs/hour-of-code/educators-2021.html

```diff
@@ -398,12 +398,93 @@ <h5 class="display-font">Assign in Teams</h5>
                     </div>
                 </div>
             </section>
-
+            <section class="hoc-section">
+                <h3 class="display-font title sunflower">
+                    Free Trainings and Live Lessons
+                </h3>
+                <p>
+                  Further prepare with educator trainings that will cover running
+                  the Hour of Code tutorial and Global MakeCode Arcade Gam Jam in 
+                  your classroom! Then throughout CSEW, tune in with your students to
+                  live lessons that provide tips and tricks on how to create a game
+                  MakeCode Arcade game. 
+                </p>
+                <ul class="accent-list">
```

Comment:
Remove this list of sessions - not necessary since all the sessions are listed out on the Events page anyway, and it just makes the page longer.